outputs/kubernetes.go

```diff
@@ -53,22 +56,47 @@ func (k *KubernetesClient) Init() error {
 	return nil
 }
 
+func (k KubernetesClient) prepareInputs(input map[string]string) map[string]map[string]string {
+	a := make(map[string]map[string]string)
+
+	for key, m := range k.KubeActions {
+		for id, val := range m {
+			if strings.HasPrefix(val, "event.input") {
```

Comment:
to make it a bit more readable I would suggest to use constant for `"event.input."`.

Reply:
another suggestion is calculate value in set of `if` statements and then set it in one final line like
`a[key] = map[string]string{id: calculated}`. It should make statement smaller

Reply:
thanks - handled that.